component/src/main/java/org/wso2/extension/siddhi/execution/streamingml/clustering/kmeans/KMeansIncrementalSPExtension.java

```diff
@@ -57,15 +56,15 @@
         namespace = "streamingml",
         description = "Performs K-Means clustering on a streaming data set. Data points can be of any dimension and " +
                 "the dimensionality is calculated from number of parameters. " +
-                "All data points to be processed by an instance of class Clusterer should be of the same " +
+                "All data points to be processed by an instance of class KMeansClusterer should be of the same " +
                 "dimensionality. The Euclidean distance is taken as the distance metric. " +
                 "The algorithm resembles Sequential K-Means Clustering at " +
                 "https://www.cs.princeton.edu/courses/archive/fall08/cos436/Duda/C/sk_means.htm ",
         parameters = {
                 @Parameter(
-                        name = "model.name",
-                        description = "The name for the model that is going to be created/reused for prediction",
-                        type = {DataType.STRING}
+                        name = "no.of.clusters",
+                        description = "The assumed number of natural clusters (numberOfClusters) in the data set.",
```

Comment:
why do u need (numberOfClusters) ?

Reply:
it is the internal name of the variable. not actually needed. will remove